po/es.po

```diff
@@ -0,0 +1,181 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: \n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2020-03-07 13:10+0100\n"
+"PO-Revision-Date: 2020-03-09 01:31+0100\n"
+"Language-Team: UBports Spanish Team \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 2.2.4\n"
+"Last-Translator: Josu\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"Language: es\n"
+
+#: qml/AddPage.qml:42
+msgid "Url (e.g. https://example.com)"
+msgstr "Url (p.e. https://ejemplo.com)"
```

Comment:
```suggestion
msgstr "Url (p.e. https://example.com)"
```
Let's keep https://example.com here, since this site actually exists and is safe to use for demonstration purposes.